app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,30 @@
+function sendVote(qid, votestring) {
+  $.ajax({
+    url: '/questions/' + qid + votestring,
+    type: 'POST',
+    dataType: "JSON",
+    data: {"_method": "put"}
+  })
+    .done(function(data) {
+      $('#' + qid).text(data);
+    })
+    .fail(function() {
+      alert("Cannot vote for the same question more than once.");
+    });
+}
+
+function initVote() {
+  $('.upvote-button').unbind().click(function(ev) {
```

Comment:
I don't think you need the `unbind()` here. It is used in `initUserFeedback()` because that is called every time a modal is opened, so you don't want to keep adding more and more event handlers. In this case `initVote()` is only going to be run once, when the page loads.